apps/heft/src/plugins/JestPlugin/JestPlugin.ts

```diff
@@ -145,6 +141,45 @@ export class JestPlugin implements IHeftPlugin {
     clean.properties.pathsToDelete.add(cacheFolder);
   }
 
+  private _getJestReporters(
+    heftSession: HeftSession,
+    heftConfiguration: HeftConfiguration
+  ): (string | Config.ReporterConfig)[] {
+    const config: Config.GlobalConfig = JsonFile.load(this._getJestConfigFolder(heftConfiguration));
```

Comment:
There's a `JsonFile.loadAsync` function that you may want to consider using here.